source/structuring.rst

```diff
@@ -306,15 +313,19 @@ $ref
 
 A schema can reference another schema using the ``$ref`` keyword. The
 value of ``$ref`` is a URI-reference that is resolved against the
-schema's `base-uri`. When evaluating a schema, an implementation uses
-the resolved identifier to retrieve the referenced schema and
-evaluation is continued from the retrieved schema.
+schema's `base-uri`. When evaluating a ``$ref``, an implementation
+uses the resolved identifier to retrieve the referenced schema and
+applies that schema to the instance.
```

Comment:
Is it worth pointing out here that "retrieve" does not mean "make an http request"?
I'm also not sure on the use of "retrieve", as it implies they are URLs rather than URIs.

Reply:
It's made clear in the section about how schemas are identified that schemas are generally retrieved from an internal schema data store. I know that people don't read these things linearly, but it's also not practical to repeat it every time it comes up. I'm open to other choices of wording to make it more clear, but I'm going to merge and we can update in a future PR.